src/main/controllers/helpers/AdminNotificationHelper.ts

```diff
@@ -17,7 +17,7 @@ export const getApplicationsWithTribunalOrderOrRequest = (
     for (const app of apps) {
```

Comment:
can make this slightly shorter by doing for (const app of app || []) rather than checking first if apps is not undefined or null

Reply:
Nice!